### PR TITLE
Fix a couple of things with `find-node` recipe

### PR DESCRIPTION
**Pull request by jletey**
Just some prose/typo fixes to the ["Find a node"](https://unifiedjs.com/learn/recipe/find-node/) recipe.

### CODE REVIEW

File: doc/learn/find-node.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ modified: 2020-01-10
 ### What is a node
 
 A node is a single language specific unit inside a [syntax tree][syntax-tree].
-Such as a heading in markdown, or anchor element in HTML.  In unified, nodes
+For example: a heading in markdown, or anchor element in HTML. In unified, nodes
```
Comment (wooorm) [inline doc/learn/find-node.md]:
```suggestion
For example: a heading in markdown, or anchor element in HTML.  In unified, nodes
```

File: doc/learn/find-node.md [hunk 2]
```diff
@@ -33,17 +33,17 @@ The concept of finding a node involves
 [tree traversal][tree-traversal] of a [syntax tree][syntax-tree].
 
 Unified compatible utilities should be used for finding a node.
-Utilities are functions that work with nodes.  All specifications
+Utilities are functions that work with nodes. All specifications
```
Comment (wooorm) [inline doc/learn/find-node.md]:
```suggestion
Utilities are functions that work with nodes.  All specifications
```

File: doc/learn/find-node.md [hunk 3]
```diff
@@ -33,17 +33,17 @@ The concept of finding a node involves
...
 that extend [unist][] can use the [unist utilities][unist-utils],
 but they can also have their own utilities for more specific nodes.
 
-To start finding nodes for your input your need:
+To start finding nodes for your input you'll need:
```
Comment (wooorm) [inline doc/learn/find-node.md]:
```suggestion
To start finding nodes for your input you’ll need:
```

File: doc/learn/find-node.md [hunk 4]
```diff
@@ -33,17 +33,17 @@ The concept of finding a node involves
...
 
 *   A processor (such as [`remark`][remark]).
 *   A utility of choice.
 
 For this example we use [`remark`][remark]
-and [`unist-util-find`][unist-util-find].  We want to find
+and [`unist-util-find`][unist-util-find]. We want to find
```
Comment (wooorm) [inline doc/learn/find-node.md]:
```suggestion
and [`unist-util-find`][unist-util-find].  We want to find
```

**Review by wooorm — CHANGES_REQUESTED** [review]
Thanks!
We do use 2 spaces between sentences, so these suggestions add them back!

**Comment by jletey** [comment]
> We do use 2 spaces between sentences

@wooorm Ah, my bad. I have added your suggestions 😄

**Review by wooorm — APPROVED** [review]

**Review by Murderlon — APPROVED** [review]
Thanks!

**Comment by jletey** [comment]
You're welcome @Murderlon

**Pull request merged**